src/tasks/AotCompilerTask/MonoAOTCompiler.cs

```diff
@@ -711,6 +733,24 @@ private PrecompileArguments GetPrecompileArgumentsFor(ITaskItem assemblyItem, st
             aotArgs.Add("dedup-skip");
         }
 
+        if (CollectCompiledMethods)
+        {
+            string assemblyFileName = Path.GetFileName(assembly);
```

Comment:
There is an existing local for this - `assemblyFilename`

Reply:
Replaced.